index.js

```diff
@@ -2,6 +2,11 @@ var mlib = require('ssb-msgs')
 var pull = require('pull-stream')
 var multicb = require('multicb')
 
+// shim for array.includes from es6
+Array.prototype.includes = function(item) {
+  return this.indexOf(item) !== -1
+}
```

Comment:
Hate to be picky about this, but it's not a good idea to modify global prototypes within a library. If it's not a perfect 1-1 match to es6's implementation, it could create bugs. I suggest just creating:

```js
function includes (array, item) {
...
```